Sources/Widgets/Representations/ResliceCursorContextRepresentation/index.js

```diff
@@ -208,6 +221,7 @@ const DEFAULT_VALUES = {
   axis1Name: '',
   axis2Name: '',
   viewName: '',
+  enableRotation: true,
```

Comment:
`enableRotation` is an `action`, a `verb`. It is more a name for a `function` than a `state` variable.

`rotationEnabled` seems like a better candidate.